### PR TITLE
Consume terra clinical site

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
Consume terra-clinical-site.

https://github.com/cerner/terra-ui.com/issues/18

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by Matt-Butler** [comment]
I'm gonna close this PR until we get https://github.com/cerner/terra-ui.com/pull/30 merged. I branched off of that pr so I wouldn't have a lot of merge conflicts.

File: client/app/bundles/App/components/Index.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Link } from 'react-router';
 import Base from 'terra-base';
 import Grid from 'terra-grid';
-import List from 'terra-list';
+import CorePackages from './CorePackages';
```
Comment (emilyrohrbough) [inline client/app/bundles/App/components/Index.jsx]: I like that you broke these into their own files!

File: .gitignore [hunk 2]
```diff
@@ -12,6 +12,7 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/db/*.sqlite3
```
Comment (emilyrohrbough) [inline .gitignore]: Looks like `db/development.sqlite3` and `db/test.sqlite3` files were added in this PR.

Comment (Matt-Butler) [inline .gitignore]: I generated the rails app with postgres and forgot to add this to the gitignore when I installed sqlite. This pr removes the two sqlite files. 

master:
https://github.com/cerner/terra-ui.com/tree/master/db

this branch:
https://github.com/cerner/terra-ui.com/tree/consume-terra-clinical-site/db

Comment (emilyrohrbough) [inline .gitignore]: Oh oops. I see that now! Looks good then!

Comment (Matt-Butler) [inline .gitignore]: +1 good question.

File: client/app/bundles/ActionHeader/components/Index.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <h2>Back, Previous-Next, Close</h2>
+    <ActionHeaderBackNextPreviousClose />
+    <h2>Maximize, Close</h2>
+    <ActionHeaderExpandClose />
+    <h2>Minimize, Custom Content, Close </h2>
+    <ActionHeaderMinimizeCustomClose />
+  </div>
+);
+
+export default HeaderExamples;
```
Comment (Matt-Butler) [inline client/app/bundles/ActionHeader/components/Index.jsx]: Does terra-clinical-action-header require translations?

I am receiving this in the console:
```
[React Intl] Missing message: "Terra.Clinical.ActionHeader.back" for locale: "en"
```

Comment (bjankord) [inline client/app/bundles/ActionHeader/components/Index.jsx]: I believe it has translations: https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-action-header/translations

However it looks like non are provided for the `en` locale, only the `en-US` `and en-GB`

Comment (Matt-Butler) [inline client/app/bundles/ActionHeader/components/Index.jsx]: https://github.com/cerner/terra-clinical/pull/95

Comment (Matt-Butler) [inline client/app/bundles/ActionHeader/components/Index.jsx]: That will fix the missing translation, however it is falling back to the id as the message. Is that the behavior we want? 



screen shot 2017-06-21 at 9 26 02 am

screen shot 2017-06-21 at 9 26 33 am
```
[React Intl] Cannot format message: "Terra.Clinical.ActionHeader.previous", using message id as fallback.
```

Comment (Matt-Butler) [inline client/app/bundles/ActionHeader/components/Index.jsx]: To unblock this pull request, I updated the locale to en-US. We are no longer receiving console errors.
https://github.com/cerner/terra-ui.com/pull/31/commits/d2fdafb921ddab695361597e9534b0f24499f6f0

Comment (Matt-Butler) [inline client/app/bundles/ActionHeader/components/Index.jsx]: I logged this to look at later: https://github.com/cerner/terra-clinical/issues/99

**Pull request merged**